contracts/gateway/EIP20CoGateway.sol

```diff
@@ -928,16 +925,13 @@ contract EIP20CoGateway is GatewayBase {
             _gasLimit
         );
 
-        // Get previousMessageHash.
-        bytes32 previousMessageHash = registerOutboxProcess(
+        // Register new redeem process
+        registerOutboxProcess(
```

Comment:
return `previousMessageHash` is not required anymore, it shouldn't be returned from the function